src/psyclone/dynamo0p3.py

```diff
@@ -734,24 +726,24 @@ def function_space_to(self):
         ''' Return the "to" function space for a gh_operator. This is
         the first function space specified in the metadata. Raise an
         error if this is not an operator. '''
-        if self._type in VALID_OPERATOR_NAMES:
+        if self._type in GH_VALID_OPERATOR_NAMES:
             return self._function_space1
         else:
             raise RuntimeError(
                 "function_space_to only makes sense for one of {0}, but "
-                "this is a '{1}'".format(VALID_OPERATOR_NAMES, self._type))
+                "this is a '{1}'".format(GH_VALID_OPERATOR_NAMES, self._type))
 
     @property
     def function_space_from(self):
         ''' Return the "from" function space for a gh_operator. This is
         the second function space specified in the metadata. Raise an
         error if this is not an operator. '''
-        if self._type in VALID_OPERATOR_NAMES:
+        if self._type in GH_VALID_OPERATOR_NAMES:
```

Comment:
Again, pylint doesn't like the `else` below.

Reply:
Done.